packages/storybook8/stories/Components/NotificationStack/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { NotificationStack as NotificationStackComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+
+import { controlsToAdd, defaultActiveNotifications, hiddenControl } from '../../controlsUtils';
+import { NotificationStackExample } from './snippets/NotificationStack.snippet';
+
+export { NotificationStack } from './NotificationStack.story';
+
+export const NotificationStackExampleDocsOnly = {
+  render: NotificationStackExample
+};
+
+const meta: Meta = {
+  title: 'Components/NotificationStack',
+  component: NotificationStackComponent,
+  argTypes: {
+    activeNotifications: controlsToAdd.activeNotifications,
+    maxNotificationsToShow: controlsToAdd.maxNotificationsToShow,
+    strings: hiddenControl,
+    ignorePremountNotifications: hiddenControl
+  },
+  args: {
+    activeNotifications: defaultActiveNotifications
```

Comment:
add hidden control for unwanted options <img width="992" alt="Screenshot 2024-07-25 at 3 28 34 PM" src="https://github.com/user-attachments/assets/8f11d814-4c4b-4308-a76a-a415855c056b">